Testing/functional/tests/cypress/integration/ui/pages/vaccineCard.js

```diff
@@ -0,0 +1,545 @@
+
+        select(dobYearSelector, dobYear);
+
+        select(dobMonthSelector, dobMonth);
+
+        clickVaccineCardEnterButton();
+
+        cy.get(feedbackDobIsRequiredSelector).should("be.visible");
+        cy.get(feedbackDovIsRequiredSelector).should("be.visible");
+    });
+
+    it("Vaccination Card - DOV Year and DOB not entered via Click Enter - unauthenticated user", () => {
+        const dovMonth = "June";
+        const dovDay = "15";
+
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(Cypress.env("phn"));
+
+        select(dovMonthSelector, dovMonth);
+
+        select(dovDaySelector, dovDay);
+
+        clickVaccineCardEnterButton();
+
+        cy.get(feedbackDobIsRequiredSelector).should("be.visible");
+        cy.get(feedbackDovIsRequiredSelector).should("be.visible");
+    });
+
+    it("Vaccination Card - DOV Month and DOB not entered via Click Enter - unauthenticated user", () => {
+        const dovYear = "2021";
+        const dovDay = "15";
+
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(Cypress.env("phn"));
+
+        select(dovYearSelector, dovYear);
+
+        select(dovDaySelector, dovDay);
+
+        clickVaccineCardEnterButton();
+
+        cy.get(feedbackDobIsRequiredSelector).should("be.visible");
+        cy.get(feedbackDovIsRequiredSelector).should("be.visible");
+    });
+
+    it("Vaccination Card - DOV Day and DOB not entered via Click Enter - unauthenticated user", () => {
+        const dovYear = "2021";
+        const dovMonth = "June";
+
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(Cypress.env("phn"));
+
+        select(dovYearSelector, dovYear);
+
+        select(dovMonthSelector, dovMonth);
+
+        clickVaccineCardEnterButton();
+
+        cy.get(feedbackDobIsRequiredSelector).should("be.visible");
+        cy.get(feedbackDovIsRequiredSelector).should("be.visible");
+    });
+
+    it("Vaccination Card - Partially Vaccinated 1 Dose - unauthenticated user", () => {
+        const phn = "9735353315";
+        const dobYear = "1967";
+        const dobMonth = "June";
+        const dobDay = "2";
+        const dovYear = "2021";
+        const dovMonth = "July";
+        const dovDay = "4";
+
+        cy.enableModules([
+            "Immunization",
+            vaccinationStatusModule,
+            "VaccinationStatusPdf",
+        ]);
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(phn);
+
+        select(dobYearSelector, dobYear);
+        select(dobMonthSelector, dobMonth);
+        select(dobDaySelector, dobDay);
+        select(dovYearSelector, dovYear);
+        select(dovMonthSelector, dovMonth);
+        select(dovDaySelector, dovDay);
+
+        clickVaccineCardEnterButton();
+
+        cy.get("[data-testid=formTitleVaccineCard]").should("be.visible");
```

Comment:
This code could be extracted into a function to avoid repeating it below.

Reply:
I will do this on the next PR when PHSA returns Dose data.